jenkins/scripts/regression-report.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#Usage: regression-report.sh <last number of days>
+# if anything fails, we'll abort
+set -e
+yum install -y python-dateutil python-blessings python-requests python-lxml
+
+no_of_days=$1
+git clone https://github.com/gluster/glusterfs.git
+cd glusterfs/
+python extras/failed-tests.py --html_report get-summary $no_of_days centos netbsd > regression_report.html
+(
+echo "To: gluster-devel@gluster.org";
```

Comment:
Need to add "From:" as well. I wasn't sure what would it be.
Also Should this script itself send the mail? or Talur was telling that as Post action of running a job we can send the mail.